File: src/wsb/po/dwa/sortowanie/BubbleSort.java
package wsb.po.dwa.sortowanie;

import java.util.Arrays;

public class BubbleSort {
    public static void bubbleSort(int[] arr) {
        int n = arr.length;
        for (int j = 0; j < n; j++) {
            for (int i = 0; i < n - 1; i++) {
                if (arr[i] > arr[i + 1]) {
                    int temp = arr[i];
                    arr[i] = arr[i + 1];
                    arr[i + 1] = temp;
                }
            }
        }
    }

    public static void main(String[] args) {
        int[] arr = {64, 34, 25, 12, 22, 11, 90};
        bubbleSort(arr);
        System.out.println("Posortowana tablica: " + Arrays.toString(arr));
    }
}

File: _Python/sortowanie/bubble_sort.py
def bubble_sort(arr):
    n = len(arr)
    for j in range(n):
        for i in range(n - 1):
            if arr[i] > arr[i + 1]:
                arr[i], arr[i + 1] = arr[i + 1], arr[i]  # Zamiana miejscami
    return arr

# Przykład użycia
arr = [64, 34, 25, 12, 22, 11, 90]
sorted_arr = bubble_sort(arr)
print("Posortowana tablica:", sorted_arr)
